tests/eclient/testdata/app_nonat.txt

```diff
@@ -0,0 +1,99 @@
+# Test app_nonat is verifying that we can use a switch network instance on a management port.
+
+{{define "port"}}2223{{end}}
+{{define "ssh"}}ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa -p {{template "port"}} root@{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with 2 reboots limit
+! test eden.reboot.test -test.v -timewait 10m -reboot=0 -count=2 &
+
+message 'Resetting of EVE'
+eden eve reset
+exec sleep 20
+
+message 'Creating networks'
+eden network create 10.11.12.0/24 -n indirect
+eden network create --type switch --uplink eth0 -n direct
+
+test eden.network.test -test.v -timewait 10m ACTIVATED indirect direct
+
+message 'Starting applications'
+eden pod deploy -v debug -n eclient docker://itmoeve/eclient:0.4 -p {{template "port"}}:22 --networks=indirect --networks=direct --memory=512MB
+
+message 'Waiting of running'
+test eden.app.test -test.v -timewait 30m RUNNING eclient
+
+message 'Checking accessibility'
+exec -t 5m bash wait_ssh.sh
+
+message 'Testing of network'
+exec sleep 20
+exec -t 1m bash ping.sh
+stdout '0% packet loss'
+
+message 'Switching network'
+eden pod modify eclient --networks indirect --networks=direct --acl='direct:'
+test eden.app.test -test.v -timewait 30m RUNNING eclient
+
+! exec -t 1m bash ping.sh
+stdout '100% packet loss'
+
+message 'Resource cleanng'
+eden pod delete eclient
+
+test eden.app.test -test.v -timewait 10m - eclient
+
+stop
+eden network delete direct
+eden network delete indirect
+
+test eden.network.test -test.v -timewait 10m - direct indirect
+
+stdout 'no network with direct found'
+stdout 'no network with indirect found'
+
+eden network ls
+! stdout '^direct\s'
+! stdout '^indirect\s'
+
+-- wait_ssh.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+
+for p in $*
```

Comment:
Any reason to use this loop here?

Reply:
Sure -- mentioning the ACL here will make sense, but the exact wording will need to be coordinated with the network documentation and the second is planned to be created.

Reply:
We have the inner loop here for that (`for i in seq 20`), right?